api/v1alpha1/k8sgpt_types.go

```diff
@@ -35,13 +35,14 @@ type K8sGPTSpec struct {
 	Backend `json:"backend"`
 	BaseUrl string `json:"baseUrl,omitempty"`
 	// +kubebuilder:default:=gpt-3.5-turbo
-	Model    string     `json:"model,omitempty"`
-	Engine   string     `json:"engine,omitempty"`
-	Secret   *SecretRef `json:"secret,omitempty"`
-	Version  string     `json:"version,omitempty"`
-	EnableAI bool       `json:"enableAI,omitempty"`
-	NoCache  bool       `json:"noCache,omitempty"`
-	Filters  []string   `json:"filters,omitempty"`
+	Model     string     `json:"model,omitempty"`
+	Engine    string     `json:"engine,omitempty"`
+	Secret    *SecretRef `json:"secret,omitempty"`
+	Version   string     `json:"version,omitempty"`
+	EnableAI  bool       `json:"enableAI,omitempty"`
+	NoCache   bool       `json:"noCache,omitempty"`
+	Filters   []string   `json:"filters,omitempty"`
+	Backstage bool       `json:"backstage,omitempty"`
```

Comment:
I dont think having this as a top level object in the type makes sense, what do you think about:

```
integrations:
 backstage: 
   enabled: true
```

Reply:
new structure is

 ```hs
 extraOptions:  
   enableBackstage: true